src/orchestrator-tracing/src/lib.rs

```diff
@@ -80,7 +83,8 @@ pub struct TracingCliArgs {
         default_value = "info"
     )]
     pub log_filter: SerializableTargets,
-    /// An optional prefix for each stderr log line.
+    /// An optional prefix for each stderr log line. Only used if
+    /// --stderr-log-format is `text`.
     #[clap(long, env = "LOG_INCLUDE_SERVICE_NAME")]
```

Comment:
We should enforce the constraint with clap! Also would be great to fix up the env var while you're in here:

```suggestion
    #[clap(long, env = "LOG_PREFIX", requires_if("log-format", "text"))]
```

Reply:
Never mind. `requires_if` doesn't work like I think it does.